src/main/java/com/exasol/errorcodecrawlermavenplugin/Finding.java

```diff
@@ -0,0 +1,26 @@
+package com.exasol.errorcodecrawlermavenplugin;
+
+/**
+ * This class represents issues with the error codes.
+ */
+public class Finding {
```

Comment:
I find the name a little bit ambiguous. `Finding` can be anything? or it's just meant to be for errors? Would make sense to rename it to something more meaningful?

Reply:
diificult, since it's errors during crawling and validation. So `ValidationFindign` does not really fit, because it can also contain errors from crawling.

Any idea?